packages/sdk-btc/source/transaction.service.ts

```diff
@@ -11,9 +12,17 @@ import { AddressFactory, BipLevel, Levels } from "./address.factory";
 import { UnspentTransaction } from "./contracts";
 import { firstUnusedAddresses, getAddresses, getDerivationMethod, post } from "./helpers";
 import { addressGenerator } from "./address.domain";
+import { LedgerService } from "./ledger.service";
+import { serializeTransaction as serializer } from "@ledgerhq/hw-app-btc/lib/serializeTransaction";
+import LedgerTransportNodeHID from "@ledgerhq/hw-transport-node-hid-singleton";
+import { ledger } from "../test/fixtures/ledger";
+import { fromPublicKey } from "bip32";
```

Comment:
https://github.com/PayvoHQ/sdk/pull/219/files#diff-4c74c4a3a6a6b26286617e21aea03cfca4890f62806e6eb91e120e3cb2f04019R1

Reply:
I know, that's because our BIP32 one didn't have the network param. Changing now.